Code-the-Dream-School/ee-prac-team3-front
# Feature/connect backend quiz data

## Purpose:
To connect backend quizz data with front end quiz cards and quiz app UI.

## Summary:
Updated front end to:
- Fetch quiz data from backend
- Store quiz data in quiz context
- Access quiz data in Quiz component to generate quiz cards
- Access quiz data in QuizContent component to generate dynamic questions and answers

## Front End Change:
App.jsx
QuizContent.jsx
QuestionContent.jsx
QuizContextProvider.jsx
useQuiz.js
index.js
exportFunctions.js

## Test Steps:
Can only be used with a working backend, which only exists on my local machine (I can't push to the backend repository)  I created my working backend by merging the working userAuthentication branch (all user calls work on this branch) with the quiz/questions-answers branch (all quiz related calls work on this branch).
Reviewer can login with email: "bear@email.com" and password: "1234", reviewer should be able to navigate to the quizzes page by clicking the quizzes button.  Then user should be able to see quizzes loaded from the back end (all quizzes are very similar right now).  User should then be able to click on card and be navigated to a quiz with unique URL based on the quiz ID.  User should be able to take the quiz and get feedback on each question (right or wrong) user should also be able to submit the test and see how they did.
